security-labs/general/creating-self-signed-cert-on-windows.md

```diff
@@ -49,6 +49,14 @@ In this exercise, you will create a self-signed certificate using Command Prompt
    ```
    
    where you replace `[your_name]` with your first and last names separated by underscore.
+  
+  - If you run into the error "unable to find distinguished_name in the config" follow these additional steps. 
```

Comment:
Those are too complex instructions and it is just one troubleshooting step. Could you please remove? I will rework the lab with WSL

Reply:
Sounds good, I removed the changes and removed the configuration file too.